src/presenters/screenViews/DoctorScreenView.java

```diff
@@ -266,4 +280,21 @@ public void showDeleteAbsenceNotAnIntegerError() {
         showDeleteNotAnIntegerError("report");
     }
 
+    public void viewReport(ReportData report) {
+        infoMessage(reportView.viewFull(report));
+
+    }
+
+    public void viewAllReports(ArrayList<ReportData> reportData) {
+        infoMessage(reportView.viewAsEnumerationFromList(reportData, reportView::viewHeader));
+    }
+
+    public Integer viewReportPrompt() {
+        return inputInt("Enter the index of the report to view in detail ");
+    }
+
+    public void noReports() {
+        infoMessage("No reports to view, please add a report");
```

Comment:
This should be an error message.

Reply:
I don't believe this qualifies as an error. We are requesting to view all reports and there just happen to be none there. This is a not an un-desired behaviour.